Library/Formula/supersonic.rb

```diff
@@ -0,0 +1,26 @@
+class Supersonic < Formula
+  homepage "https://code.google.com/p/supersonic/"
+  url "https://supersonic.googlecode.com/files/supersonic-0.9.4.tar.gz"
+  sha256 "1592dfd2dc73f0b97298e0d25e51528dc9a94e9e7f4ab525569f63db0442d769"
+
+  depends_on :x11
+  depends_on "glog"
+  depends_on "protobuf"
+  depends_on "pkg-config" => :build
+
+  def install
+    args=[]
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--without-re2"
+    system "make", "clean"
+    system "make", "check"
+    system "make", "install"
+  end
+
+  test do
+    system "false"
```

Comment:
Can you add a test that tests the application's functionality? The info on what we like to see is [in the formula cookbook](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#add-a-test-to-the-formula).